.github/pull_request_template.md

```diff
@@ -1,48 +1,31 @@
 ## Changes
 
-_Explanation of changes goes here_
-
-## Notes
-
-_Any other notes go here_
+Changed the save/cancel buttons in the RiskLog component to use the NERSuccessButton/NERFailButton component rather than just the plain MUI button
 
 ## Test Cases
 
-- Case A
-- Edge case
-- ...
+- yarn test:frontend
 
 ## Screenshots
 
-_If you made UI changes you must post a screenshot of the whole page for each change in 1) a normal sized window and 2) the smallest possible window_
-
-_If you did any manual testing (e.g., with Postman), put screenshots of the http request and before and after of the db_
-
-_If none of this applies, you can delete this section_
-
-## To Do
-
-_Any remaining things that need to get done_
-
-- [ ] item 1
-- [ ] ...
+image.png
 
 ## Checklist
 
 It can be helpful to check the `Checks` and `Files changed` tabs.
 Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
 Please request reviewers and ping on slack only after you've gone through this whole checklist.
 
-- [ ] All commits are tagged with the ticket number
-- [ ] No linting errors / newline at end of file warnings
-- [ ] All code follows repository-configured prettier formatting
-- [ ] No merge conflicts
-- [ ] All checks passing
-- [ ] Screenshots of UI changes (see Screenshots section)
-- [ ] Remove any non-applicable sections of this template
-- [ ] Assign the PR to yourself
-- [ ] No `yarn.lock` changes (unless dependencies have changed)
-- [ ] Request reviewers & ping on Slack
-- [ ] PR is linked to the ticket (fill in the closes line below)
-
-Closes # (issue #)
+- [x] All commits are tagged with the ticket number
+- [x] No linting errors / newline at end of file warnings
+- [x] All code follows repository-configured prettier formatting
+- [x] No merge conflicts
+- [x] All checks passing
+- [x] Screenshots of UI changes (see Screenshots section)
+- [x] Remove any non-applicable sections of this template
+- [x] Assign the PR to yourself
+- [x] No `yarn.lock` changes (unless dependencies have changed)
+- [x] Request reviewers & ping on Slack
+- [x] PR is linked to the ticket (fill in the closes line below)
+
```

Comment:
you should undo all the changes to this file, not sure how it got in here, probably just some github weirdness/ not your bad